packages/ui/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { forwardRef, HTMLChakraProps, Radio as ChakraRadio } from '@chakra-ui/react';
+
+export type RadioSize = 'md' | 'lg';
+
+export interface RadioProps extends Omit<HTMLChakraProps<'div'>, 'onChange' | 'defaultChecked'> {
+  /**
+   * Determines radio's and text size.
+   */
+  size?: RadioSize;
+  /**
+   * Determines state of the radio.
+   */
+  checked?: boolean;
+  /**
+   * Determines if input is disabled.
+   */
+  disabled?: boolean;
+  value: string | number;
+}
+
+export const Radio = forwardRef<RadioProps, 'input'>(
+  ({ value, checked = false, disabled = false, size = 'md', ...props }, ref) => {
+    return (
+      <ChakraRadio
```

Comment:
A small thing, but Radio should have a different color when use is clicking it (active state, third row on mockups). You can use [theme component syles](https://chakra-ui.com/docs/theming/component-style), to apply it